### PR TITLE
New config for nrsecurityagent

### PR DESCRIPTION
**Pull request by aayush-ap**
Added a new config to support the new feature release in CSEC Agent Release v1.5.0

### CODE REVIEW

File: v3/integrations/nrsecurityagent/nrsecurityagent.go [hunk 1]
```diff
@@ -63,7 +69,7 @@ func InitSecurityAgent(app *newrelic.Application, opts ...ConfigOption) error {
 
 	appConfig, isValid := app.Config()
 	if !isValid {
-		return fmt.Errorf("Newrelic application value cannot be read; did you call newrelic.NewApplication?")
+		return fmt.Errorf("Newrelic  application value cannot be read; did you call newrelic.NewApplication?")
```
Comment (nr-swilloughby) [inline v3/integrations/nrsecurityagent/nrsecurityagent.go:72]: Is that a typo or do you intend the extra space to be added to the error string?

**Review by nr-swilloughby — CHANGES_REQUESTED** [review]
just one quick question, otherwise it looks good.

File: v3/integrations/nrsecurityagent/nrsecurityagent.go (hunk 1 above)
Comment (aayush-ap) [reply v3/integrations/nrsecurityagent/nrsecurityagent.go:72]: yes extra space is typo.

Comment (nr-swilloughby) [reply v3/integrations/nrsecurityagent/nrsecurityagent.go:72]: ok we can fix it on release.

**Review by nr-swilloughby — APPROVED** [review]

**Pull request merged**